lib/src/model.dart

```diff
@@ -3213,9 +3219,51 @@ class ModelFunction extends ModelFunctionTyped {
   }
 
   @override
+  String get name {
+    if (element.enclosingElement is ParameterElement && super.name.isEmpty)
+      return element.enclosingElement.name;
```

Comment:
I'm not sure of the context of this code, so it might not matter, but parameters are no longer required to have a name (when inside a generic function type).

Reply:
Yes, that's OK here.  With this CL, it's now more OK than it used to be, in fact.